rubin_scheduler/utils/consdb.py

```diff
@@ -305,6 +309,10 @@ def consdb_visits(self) -> pd.DataFrame:
         # and visit_id = exposure_id.
         # To support 2 exposures snaps in the future, this query will
         # need to be rewritten to group by visit_id.
+        day_obs_date = datetime.strptime(f"{self.day_obs_int}", "%Y%m%d").date()
+        prior_day_obs_date = day_obs_date - timedelta(days=self.num_nights)
+        prior_day_obs = int(prior_day_obs_date.strftime("%Y%m%d"))
```

Comment:
I find it much easier to keep track of what format is what for dayobs or dayobsint (or other numbers that are into or out of that format) if I always use the same "kind" of names. 
As in -- if prior_day_obs is the same kind of thing as day_obs_int, couldn't it be "prior_day_obs_int"? 
Otherwise though, this seems fine.